e2e/portal/tests/ChangeRegistrationStatus/DeleteRegistrationWithStatusCompleted.spec.ts

```diff
@@ -23,8 +23,8 @@ const toastMessage =
 
 // Arrange
 test.beforeEach(async ({ page }) => {
-  const accessToken = await getAccessToken();
   await resetDB(SeedScript.nlrcMultiple, __filename);
+  const accessToken = await getAccessToken();
```

Comment:
Well.. why did this even work in the first place?? :D

Getting the access-token from BEFORE resetting the DB should not "still work". ;)